tools/distrib/python/grpcio_tools/grpc_version.py

```diff
@@ -29,4 +29,4 @@
 
 # AUTO-GENERATED FROM `$REPO_ROOT/templates/tools/distrib/python/grpcio_tools/grpc_version.py.template`!!!
 
-VERSION='0.15.0'
+VERSION='0.15.1'
```

Comment:
I see "AUTO-GENERATED FROM..." on the above line. It looks like if you want to change this, you should change build.yaml and update the version for everything.